yarn_api_client/resource_manager.py

```diff
@@ -14,19 +14,41 @@ class ResourceManager(BaseYarnAPI):
     and information about applications on the cluster.
 
     If `address` argument is `None` client will try to extract `address` and
-    `port` from Hadoop configuration files.
+    `port` from Hadoop configuration files.  If both `address` and `alt_address`
+    are provided, the address corresponding to the ACTIVE HA Resource Manager will
+    be used.
 
     :param str address: ResourceManager HTTP address
     :param int port: ResourceManager HTTP port
     :param int timeout: API connection timeout in seconds
     :param boolean kerberos_enabled: Flag identifying is Kerberos Security has been enabled for YARN
+    :param dict: additional arguments, including...
+        str alt_address: ResourceManager HTTP address for HA configurations
+        int alt_port: ResourceManager HTTP port for HA configuration
     """
-    def __init__(self, address=None, port=8088, timeout=30, kerberos_enabled=False):
+    def __init__(self, address=None, port=8088, timeout=30, kerberos_enabled=False, **kwargs):
```

Comment:
Can we use explicit arg versus kwargs, other than that it looks good for now.

Reply:
Will do - thanks.